packages/angular/src/generators/ngrx/lib/add-imports-to-module.ts

```diff
@@ -0,0 +1,198 @@
+import type { Tree } from '@nrwl/devkit';
+import { names } from '@nrwl/devkit';
+import { insertImport } from '@nrwl/workspace/src/utilities/ast-utils';
+import type { SourceFile } from 'typescript';
+import { createSourceFile, ScriptTarget } from 'typescript';
+import {
+  addImportToModule,
+  addProviderToModule,
+} from '../../../utils/nx-devkit/ast-utils';
+import type { NgRxGeneratorOptions } from '../schema';
+
+export function addImportsToModule(
+  tree: Tree,
+  options: NgRxGeneratorOptions
+): void {
+  const modulePath = options.module;
+  const sourceText = tree.read(modulePath, 'utf-8');
+  let sourceFile = createSourceFile(
+    modulePath,
+    sourceText,
+    ScriptTarget.Latest,
+    true
+  );
+  const addImport = (
```

Comment:
It is and it was there from before, I kept it but I don't have a strong opinion on whether to have it or not.

On the other hand, I personally don't see the need to add a comment, the name of the utility says what it does and it's really straightforward, there's no logic inside making it hard to understand what's happening, it's just a convenience to avoid some repetition. I'm of the opinion comments should only be added when really needed otherwise they are just noise. I get the "really needed" part is subjective 🙂 imho I don't think it is needed here.

Reply:
No problem!